File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/inputs/inputWithExtendedType/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.inputs.inputWithExtendedType.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String Movies = "movies";

    public static class MOVIES_INPUT_ARGUMENT {
      public static final String Filter = "filter";
    }
  }

  public static class MOVIEFILTER {
    public static final String TYPE_NAME = "MovieFilter";

    public static final String Genre = "genre";

    public static final String ReleaseYear = "releaseYear";
  }
}

File: graphql-dgs-codegen-core/src/test/resources/kotlin2/inputWithExtendedType/expected/DgsConstants.kt
package kotlin2.inputWithExtendedType.expected

import kotlin.String

public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val Movies: String = "movies"

    public object MOVIES_INPUT_ARGUMENT {
      public const val Filter: String = "filter"
    }
  }

  public object MOVIEFILTER {
    public const val TYPE_NAME: String = "MovieFilter"

    public const val Genre: String = "genre"

    public const val ReleaseYear: String = "releaseYear"
  }
}
